src/components/content/order/orderStatus/OrderProcessingStatus.tsx

```diff
@@ -113,5 +113,26 @@ export const OrderProcessingStatus = ({
             );
         }
     }
+
+    if (operationType === OperationType.SubmitAction) {
+        if (serviceOrderStatus.taskStatus === 'successful') {
+            return (
+                <div>
+                    <span>{'Service action request submitted successfully.'}</span>
```

Comment:
is it possible to also show the 'action name' in bold in the message.
for example - 'Service action backup request submitted successfully'